--- y-ozel-hastaneler (eski 3a2c222e · RG 30.01.2025/32798)
+++ y-ozel-hastaneler (yeni 59919bf9 · RG 30.01.2025/32798)
@@ -2,75 +2,76 @@
 
                                                                          BİRİNCİ BÖLÜM
                                                                         Başlangıç Hükümleri
-        Amaç
-        MADDE 1- (1) Bu Yönetmeliğin amacı; etkin, verimli ve kaliteli sağlık hizmeti sunulmasını sağlamak üzere, özel hastanelerin hizmet, tıp
+         Amaç
+         MADDE 1- (1) Bu Yönetmeliğin amacı; etkin, verimli ve kaliteli sağlık hizmeti sunulmasını sağlamak üzere, özel hastanelerin hizmet, tıp
 teknolojisi, tesis ve personel standartlarının belirlenmesine, teşkilatlandırılmasına, planlamasına ve bunların açılmalarına, faaliyetlerine, kapatılmalarına ve
 denetimlerine ilişkin usul ve esasları düzenlemektir.
-        Kapsam
-        MADDE 2- (1) Bu Yönetmelik; ağız ve diş sağlığı hastaneleri hariç olmak üzere gerçek kişiler veya özel hukuk tüzel kişilerine ait hastaneleri
+         Kapsam
+         MADDE 2- (1) Bu Yönetmelik; ağız ve diş sağlığı hastaneleri hariç olmak üzere gerçek kişiler veya özel hukuk tüzel kişilerine ait hastaneleri
 kapsar.
-        Dayanak
-        MADDE 3-(1) Bu Yönetmelik; 24/5/1933 tarihli ve 2219 sayılı Hususi Hastaneler Kanunu hükümlerine, 7/5/1987 tarihli ve 3359 sayılı Sağlık
+         Dayanak
+         MADDE 3-(1) Bu Yönetmelik; 24/5/1933 tarihli ve 2219 sayılı Hususi Hastaneler Kanunu hükümlerine, 7/5/1987 tarihli ve 3359 sayılı Sağlık
 Hizmetleri Temel Kanununun 3 üncü maddesi, 9 uncu maddesinin (c) bendi, ek 11 inci, ek 15 inci ve ek 19 uncu maddeleri ile 1 sayılı Cumhurbaşkanlığı
 Teşkilatı Hakkında Cumhurbaşkanlığı Kararnamesinin 355 inci maddesinin birinci fıkrasının (a) ve (c) bentleri ile 508 inci maddesine dayanılarak
 hazırlanmıştır.
-        Tanımlar
-        MADDE 4- (1) Bu Yönetmelikte geçen;
-        a) Bakanlık: Sağlık Bakanlığını,
-        b) EKİP: Entegre Kurumsal İşlem Platformunu,
-        c) Faaliyet izin belgesi: Ek-7’de örneğine yer verilen ve ruhsatlandırılmış hastanelerin faaliyete geçebilmeleri için veya faaliyette olan
+         Tanımlar
+         MADDE 4- (1) Bu Yönetmelikte geçen;
+         a) Bakanlık: Sağlık Bakanlığını,
+         b) EKİP: Entegre Kurumsal İşlem Platformunu,
+         c) Faaliyet izin belgesi: Ek-7’de örneğine yer verilen ve ruhsatlandırılmış hastanelerin faaliyete geçebilmeleri için veya faaliyette olan
 hastanelerin hasta kabul ve tedavi edeceği uzmanlık dallarını, hizmet verilen laboratuvarları, Bakanlıkça özellikli planlama kapsamına alınan tıbbi
 cihazları, diğer tıbbi hizmet birimlerini ve yatak kapasitesi ile bunlardaki değişikliklerin yer aldığı il sağlık müdürlüğünce düzenlenen belgeyi,
-        ç) Genel Müdür: Sağlık Hizmetleri Genel Müdürünü,
-        d) Genel Müdürlük: Sağlık Hizmetleri Genel Müdürlüğünü,
-        e) Müdürlük: İl veya ilçe sağlık müdürlüğünü,
-        f) Özel hastane: Gerçek kişiler veya özel hukuk tüzel kişileri adına Bakanlıkça ruhsatlandırılan, her türlü acil vaka ile bünyesinde mevcut
+         ç) Genel Müdür: Sağlık Hizmetleri Genel Müdürünü,
+         d) Genel Müdürlük: Sağlık Hizmetleri Genel Müdürlüğünü,
+         e) Müdürlük: İl veya ilçe sağlık müdürlüğünü,
+         f) Özel hastane: Gerçek kişiler veya özel hukuk tüzel kişileri adına Bakanlıkça ruhsatlandırılan, her türlü acil vaka ile bünyesinde mevcut
 uzmanlık dallarıyla ilgili hastaların kabul edildiği, hastalarının ayaktan veya yatarak müşahede, muayene, teşhis, tedavi ve rehabilitasyonlarının yapıldığı,
 ruhsatlarında ve faaliyet izin belgelerinde yazılı tür ve sayıdaki hizmet birimleriyle çalışan yataklı sağlık kurumlarını,
-        g) Ruhsat: Özel hastanelerin sağlık hizmet sunumuna uygunluğunu gösteren, açılışta hastane sahibi adına düzenlenen ve hastane isim, adres ve
+         g) Ruhsat: Özel hastanelerin sağlık hizmet sunumuna uygunluğunu gösteren, açılışta hastane sahibi adına düzenlenen ve hastane isim, adres ve
 yatak sayısının yer aldığı Bakanlıkça düzenlenen belgeyi,
-        ğ) Sağlık personeli: Tabip, diş tabibi, eczacı, hemşire, optisyen ve 11/4/1928 tarihli ve 1219 sayılı Tabâbet ve Şu’abatı San’atlarının Tarz-ı
+         ğ) Sağlık personeli: Tabip, diş tabibi, eczacı, hemşire, optisyen ve 11/4/1928 tarihli ve 1219 sayılı Tabâbet ve Şu’abatı San’atlarının Tarz-ı
 İcrâsına Dâir Kanunda tanımlanan diğer sağlık meslek mensuplarını,
-        h) Uzmanlık dalı: Tıpta uzmanlık mevzuatında belirlenmiş olan uzmanlık alanlarını,
-        ifade eder.
+         h) Uzmanlık dalı: Tıpta uzmanlık mevzuatında belirlenmiş olan uzmanlık alanlarını,
+         ifade eder.
                                                                            İKİNCİ BÖLÜM
-                                            Özel Hastanelerin Temel Özellikleri, Hizmet Birimleri ve Nitelikleri,
+                                             Özel Hastanelerin Temel Özellikleri, Hizmet Birimleri ve Nitelikleri,
                                                         Özel Hastane Açmaya ve İşletmeye Yetkili Kişiler
-        Özel hastanelerin temel özellikleri
-        MADDE 5- (1) Özel hastaneler, bu Yönetmelikte asgari olarak öngörülen bina, hizmet, tıbbi cihaz ve personel standartlarını haiz olmak kaydıyla,
+         Özel hastanelerin temel özellikleri
+         MADDE 5- (1) Özel hastaneler, bu Yönetmelikte asgari olarak öngörülen bina, hizmet, tıbbi cihaz ve personel standartlarını haiz olmak kaydıyla,
 yirmi dört saat süreyle sürekli ve düzenli olarak, bünyesindeki mevcut uzmanlık dallarıyla hastalara ayakta ve yatarak muayene, teşhis ve tedavi hizmeti
 verir.
-        (2) Özel hastaneler; bu Yönetmelik hükümlerine göre sağlık hizmet sunumu açısından projesi Bakanlıkça onaylanmış ve Ek-5’te belirtilen fiziki
+         (2) Özel hastaneler; bu Yönetmelik hükümlerine göre sağlık hizmet sunumu açısından projesi Bakanlıkça onaylanmış ve Ek-5’te belirtilen fiziki
 şartları sağlayan müstakil binalarda hizmet verir.
-        (3) Özel hastaneler gözlem yatakları hariç en az yüz hasta yatağı ile açılır. Bakanlıkça belirlenen sağlık hizmet bölgelerine göre planlanan tabip
+         (3) Özel hastaneler gözlem yatakları hariç en az yüz hasta yatağı ile açılır. Bakanlıkça belirlenen sağlık hizmet bölgelerine göre planlanan tabip
 sayısı ve hizmet ihtiyacı doğrultusunda en az elli hasta yataklı olmak üzere yüz yatağın altında özel hastane açılmasına izin verilebilir. Özel hastanelerin
 yoğun bakım yatak sayısının toplamı, ruhsata esas toplam hasta yatak sayısının yüzde 30’unu geçemez.
-        (4) Birbirine bitişik ve bütünlük arz eden birden fazla imar parseli üzerinde hastane ve eklerinin yapılması halinde, bu parseller arasında imar yolu,
+         (4) Birbirine bitişik ve bütünlük arz eden birden fazla imar parseli üzerinde hastane ve eklerinin yapılması halinde, bu parseller arasında imar yolu,
 duvar gibi bütünlüğü bozacak uygulamalar bulunamaz.
-        (5) Aynı sahiplikte ve aynı isim altında ruhsatlandırılmak kaydıyla, sınırları belirli, bütünlük arz eden bir alan içinde birbiriyle fiziki olarak bağlantısı
+         (5) Aynı sahiplikte ve aynı isim altında ruhsatlandırılmak kaydıyla, sınırları belirli, bütünlük arz eden bir alan içinde birbiriyle fiziki olarak bağlantısı
 olan veya bağımsız bina komplekslerinin oluşturduğu hastane binalarından oluşan hastane kompleksi kurulabilir. Bu durumda, laboratuvar hizmetleri,
 görüntüleme hizmetleri, acil hizmetleri, ameliyathaneler, yoğun bakım ve sterilizasyon üniteleri ile benzeri diğer tıbbi hizmetler ortak hizmet verecek
 şekilde planlanabilir. İdari birimler, morg, depo ve benzeri destek birimler ile otopark, kreş, kafeterya, lokanta, otel, banka, PTT, konferans salonu,
 ibadethane, spor ve eğitim tesisleri gibi sosyal alanlar hizmeti aksatmayacak şekilde ve mimari projeye işlenerek onaylanmak kaydıyla yapı kullanma
 izni alınarak hastaneye ait belirlenen sınırlar içerisinde farklı binalarda hizmet verebilir.
-        (6) Hastane personeline, hastalara ve hasta yakınlarına hizmet vermek ve işletilmek üzere kafeterya, kantin, çiçekçi, berber, lostra gibi birimler
+         (6) Hastane personeline, hastalara ve hasta yakınlarına hizmet vermek ve işletilmek üzere kafeterya, kantin, çiçekçi, berber, lostra gibi birimler
 kurulabilir veya bunların işletilmesi için hizmet satın alınabilir veyahut bu hizmetler başkalarına gördürülebilir. Bu türden kısımlar, tıbbi hizmet birimlerinin
 bulunduğu yerler ile hasta tedavi ve istirahatini engelleyecek yerlerde kurulamaz.
-        (7) Hastane binası içinde hiçbir surette başka amaca yönelik işyeri bulunamaz ve hastane içindeki bir bölüm, altıncı fıkrada sayılan işletmeler
+         (7) Hastane binası içinde hiçbir surette başka amaca yönelik işyeri bulunamaz ve hastane içindeki bir bölüm, altıncı fıkrada sayılan işletmeler
 dışındaki bir amaçla faaliyet göstermek üzere üçüncü kişilere kiralanamaz veya herhangi bir şekilde devredilemez.
-        (8) (Danıştay Onuncu Dairesinin 18/9/2025 tarihli ve E.:2025/1597 sayılı kararı ile yürütmesi durdurulan fıkra: Özel hastaneler
-ruhsatlandırıldıkları tarihten itibaren en geç üç yıl içinde Türkiye Sağlık Hizmetleri Kalite ve Akreditasyon Enstitüsünden (TÜSKA)
-akreditasyon belgesi almak zorundadır. Akreditasyon belgesi alamayan hastanelerde, hastaların görebileceği şekilde “Bu hastanenin
-TÜSKA akreditasyon belgesi yoktur” ibareli Ek-14’te belirlenen standartlara uygun bilgilendirme tabelası asılmak zorundadır. Bakanlıkça
-bu hastaneler her yıl ocak ayında Sosyal Güvenlik Kurumuna bildirilir.)
-        Özel hastanelerin hizmet birimleri ve nitelikleri
-        MADDE 6- (1) Özel hastanelerde bulunan tüm birimler Ek-5’te belirtilen asgari fiziki kriterleri sağlamak zorundadır.
-        (2) Bu Yönetmelikte belirlenmiş asgari fiziki standartlara ilişkin ölçülerde %2’ye kadar sapmalara izin verilir.
-        (3) Özel hastanelerde bulunan gözlem yatak sayısı, hastane toplam yatak sayısının %25’ini geçemez. Bir gözlem ünitesinde en fazla yirmi yatak
+         (8) (Değişik:RG-29/8/2026-33355) Türkiye Sağlık Hizmetleri Kalite ve Akreditasyon Enstitüsü (TÜSKA) akreditasyon belgesi alan özel
+hastanelerden, 31/12/2016 tarihli ve 29935 üçüncü mükerrer sayılı Resmî Gazete’de yayımlanan Türkiye Sağlık Hizmetleri Kalite ve Akreditasyon
+Enstitüsünün Yapılanması ve Faaliyetlerinin Yürütülmesine Dair Yönetmeliğe dayanılarak TÜSKA tarafından belirlenen usul ve esaslar uyarınca platin
+veya altın kategorisine göre belgelendirilenler 3 yıl, gümüş kategorisine göre belgelendirilenler 2 yıl, bronz kategorisine göre belgelendirilenler ise 1 yıl
+boyunca olağan denetimlerden muaf tutulur.
+         Özel hastanelerin hizmet birimleri ve nitelikleri
+         MADDE 6- (1) Özel hastanelerde bulunan tüm birimler Ek-5’te belirtilen asgari fiziki kriterleri sağlamak zorundadır.
+         (2) Bu Yönetmelikte belirlenmiş asgari fiziki standartlara ilişkin ölçülerde (Değişik ibare:RG-29/8/2026-33355) %5’e kadar sapmalara izin
+verilir.
+         (3) Özel hastanelerde bulunan gözlem yatak sayısı, hastane toplam yatak sayısının %25’ini geçemez. Bir gözlem ünitesinde en fazla yirmi yatak
 bulunabilir.
-        (4) Yalnızca dahili uzmanlık dallarında hizmet verecek olan hastanelerde ameliyathane bulunması zorunlu değildir. Bu şekilde açılan hastanelerde,
+         (4) Yalnızca dahili uzmanlık dallarında hizmet verecek olan hastanelerde ameliyathane bulunması zorunlu değildir. Bu şekilde açılan hastanelerde,
+ cerrahi uzmanlık dalı ilavesi, bu Yönetmelikte belirtilen ameliyathane ve yoğun bakım ile diğer şartların sağlanması durumunda yapılır.
-         (5) Ameliyathanenin işleyişi, hizmete devamlı hazır bulundurulması, alet ve malzemenin sağlanması, bakım, onarım ihtiyaçlarının saptanarak
 yaptırılmak üzere ilgililere bildirilmesi ve burada çalışan personelin yönetimi ve eğitimlerinin yapılması amacıyla cerrahi branşlardan bir uzman tabip
 ameliyathane sorumlusu olarak görevlendirilir.
@@ -82,13 +83,12 @@
 bakılmaksızın yalnızca bir uzman tabip bulunduğunda ilgili yoğun bakım ünitesi kurulması zorunlu değildir.
         (8) Özel hastane bünyesinde yürütülen tıbbi laboratuvar, radyoloji, radyasyon onkolojisi, genetik hastalıklar tanı ve nükleer tıp hizmetleri ilgili
 yönetmelikler kapsamında yürütülür. Bu birimler için ilgili mevzuata göre hastane adına faaliyet izin belgesi düzenlenir.
-        (9) (Danıştay Onuncu Dairesinin 31/12/2025 tarihli ve E.:2025/1584 sayılı kararı ile yürütmesi durdurulan fıkra:Özel hastanelerde,
-bir adet seyyar röntgen cihazı bulunması ve en az 500 mA gücünde röntgen cihazının bulunduğu bir radyoloji merkezi kurulması
-zorunludur.)
-        (10) (Danıştay Onuncu Dairesinin 4/2/2026 tarihli ve E.:2025/1456 sayılı kararı ile yürütmesi durdurulan fıkra:Özel hastanelerde
-hasta kabul ve tedavi ettiği uzmanlık dallarının gerektirdiği tıbbi biyokimya ve/veya tıbbi mikrobiyoloji laboratuvarı bulunması zorunludur.
-Özel hastanenin açmak istediği tıbbi biyokimya ve/veya tıbbi mikrobiyoloji laboratuvarı için uzman tabip kadrosunun bulunmaması
-durumunda, talep etmesi halinde bu uzmanlık dalında Bakanlıkça kadro verilir.)
+        (9) (Değişik:RG-29/8/2026-33355) Özel hastanelerde, bir adet seyyar röntgen cihazı bulunması veya en az 500 mA gücünde röntgen cihazı
+bulunması zorunludur. Röntgen cihazının, yerinde hizmet alımı yoluyla özel hastanede bulundurulması halinde bu zorunluluk yerine getirilmiş sayılır.
+        (10) (Değişik:RG-29/8/2026-33355) Özel hastanelerde hasta kabul ve tedavi ettiği uzmanlık dallarının gerektirdiği tıbbi biyokimya ve/veya
+tıbbi mikrobiyoloji laboratuvarı bulunması zorunludur. Belirli bir alanda sağlık hizmeti sunan özel hastaneler, bu zorunluluğu yerinde hizmet alımı yoluyla
+da yerine getirebilir. Özel hastanenin açmak istediği tıbbi biyokimya ve/veya tıbbi mikrobiyoloji laboratuvarı için uzman tabip kadrosunun bulunmaması
+durumunda, talep etmesi halinde bu uzmanlık dalında Bakanlıkça kadro verilir.
         (11) Özel hastanelerin bünyesinde yapılmayan, radyoloji ve tıbbi laboratuvar tetkikleri için hizmet alımı yapılabilir. Özel hastaneler bu hizmet
 alımlarını ruhsat/faaliyet izin belgesi bulunan diğer sağlık kuruluşlarından veya tıbbi cihaz satış yetki belgesine sahip yüklenici firmalardan yerinde hizmet
 alımı şeklinde yapabilir. Bu durumda hizmet satın alınmasına ait belgeler faaliyet iznine esas belgeler ile birlikte Müdürlüğe sunulur.
@@ -240,11 +240,10 @@
 1219 sayılı Kanunun 12 nci maddesine uygun olmak kaydıyla aynı ilde kadrolu çalıştığı özel sağlık kuruluşu dışında planlama kapsamındaki bir özel
 sağlık kuruluşunda daha kadro dışı geçici çalışabilir.
          (4) Özel hastanelerde çalışan tabip dışı sağlık personeli, en fazla bir özel sağlık kuruluşunda daha çalışabilir.
-         (5) (Danıştay İDDK’nın 9/4/2026 tarihli ve E.:2026/231 sayılı kararı ile yürütmesi durdurulan ibare:Vakıf üniversiteleriyle iş birliği
-protokolü bulunan) özel hastaneler, tabip ve diş tabibi kadro sayıları için ayrı ayrı hesaplanmak şartıyla ve bu kadro sayılarının yüzde yirmisini
-geçmemek üzere belirlenen sayıda profesör ve doçenti, 4/11/1981 tarihli ve 2547 sayılı Yükseköğretim Kanununun 36 ncı maddesinin yedinci fıkrası
-kapsamında Devlet üniversitelerince yapılacak değerlendirmede uygun bulunması halinde yapacakları kurumsal sözleşme çerçevesinde mesai dışında
-hastanenin boş uzman tabip kadrosunda çalıştırabilir.
+         (5) (Değişik ibare:RG-29/8/2026-33355) Bakanlıkça onaylanmış uzmanlık eğitim programı bulunan özel hastaneler, tabip ve diş tabibi kadro
+sayıları için ayrı ayrı hesaplanmak şartıyla ve bu kadro sayılarının yüzde yirmisini geçmemek üzere belirlenen sayıda profesör ve doçenti, 4/11/1981
+tarihli ve 2547 sayılı Yükseköğretim Kanununun 36 ncı maddesinin yedinci fıkrası kapsamında Devlet üniversitelerince yapılacak değerlendirmede
+uygun bulunması halinde yapacakları kurumsal sözleşme çerçevesinde mesai dışında hastanenin boş uzman tabip kadrosunda çalıştırabilir.
          (6) Diyaliz merkezleri ve üremeye yardımcı tedavi merkezlerinde ilgili mevzuatı uyarınca kadrolu çalışan uzman tabipler, üçüncü fıkraya uygun
 olmak kaydıyla özel hastanede çalışabilir.
          (7) Özel sağlık kuruluşlarında çalışan tabip/uzman tabipler, 1219 sayılı Kanunun 12 nci maddesine uygun olmak kaydıyla, çalıştıkları sağlık
@@ -264,11 +263,10 @@
 tabipler hariç olmak üzere, yurtdışında en az iki yıl süreyle mesleğini icra eden tabipler, yurtdışından döndükten sonra altı ay içinde başvurması halinde
 Türkiye’de çalışmak üzere tercih edecekleri özel hastanede bir kereye mahsus olmak üzere kadro dışı geçici çalışabilir.
          (4) Sağlık durumu aciliyet arz eden ve/veya başka kuruma sevki tıbben riskli olan hastalar için, tedavisini üstlenen tabibin talebi üzerine tıbbi
-görüş, destekleyici ve tamamlayıcı hizmet veya yardım alınması için konsültasyon hizmetine ihtiyaç duyulan uzmanlık dalında o ildeki (Danıştay
-Onuncu Dairesinin 31/12/2025 tarihli ve E.:2025/1585 sayılı kararı ile yürütmesi durdurulan ibare; öncelikle özel sağlık kuruluşlarındaki
-kadrolu tabiplerden olmak üzere ) konsültan tabip davet edilebilir. Konsültan tabip hizmet bedeli hastane tarafından hizmet sunan kuruma ödenir. Bu
-şekilde hastaneye davet edilen konsültan tabiplerin isimleri ve uzmanlık dalları hakkında mesul müdür tarafından en geç beş iş günü içinde Müdürlüğe
-bilgi verilir.
+görüş, destekleyici ve tamamlayıcı hizmet veya yardım alınması için konsültasyon hizmetine ihtiyaç duyulan uzmanlık dalında o ildeki (Değişik
+ibare:RG-29/8/2026-33355) tercihen özel sağlık kuruluşlarındaki kadrolu tabiplerden olmak üzere konsültan tabip davet edilebilir. Konsültan tabip
+hizmet bedeli hastane tarafından hizmet sunan kuruma ödenir. Bu şekilde hastaneye davet edilen konsültan tabiplerin isimleri ve uzmanlık dalları
+hakkında mesul müdür tarafından en geç beş iş günü içinde Müdürlüğe bilgi verilir.
          (5) Özel hastane, Bakanlıkça izin verilmesi kaydıyla yabancı tabip ve tabip dışı sağlık meslek mensubu çalıştırabilir. Ancak bu şekilde verilecek
 izin, hastaneye planlama kapsamında cihaz ve ünite ekleme hakkı vermez. Bu fıkra doğrultusunda yapılacak başvurular 22/2/2012 tarihli ve 28212 sayılı
 Resmî Gazete’de yayımlanan Yabancı Sağlık Meslek Mensuplarının Türkiye'de Özel Sağlık Kuruluşlarında Çalışma Usul ve Esaslarına Dair
@@ -289,9 +287,9 @@
          (4) Mesul müdür, görev yaptığı hastanenin dışında başka bir sağlık kurum ve kuruluşunda çalışamaz.
          (5) Mesul müdürün görev, yetki ve sorumlulukları şunlardır:
          a) Özel hastanenin tıbbi, idari ve teknik hizmetlerini mevzuata uygun olarak idare etmek ve denetlemek.
--        b) Ruhsatlandırma ile ilgili her türlü işlemleri yürütmek.
-        c) Özel hastanenin ruhsatlandırılmasından sonra, personel, bina ve tesis değişikliklerini Müdürlüğe bildirmek.
+         b) Ruhsatlandırma ile ilgili her türlü işlemleri yürütmek.
+         c) Özel hastanenin ruhsatlandırılmasından sonra, personel, bina ve tesis değişikliklerini Müdürlüğe bildirmek.
+         ç) Özel hastane adına belge onaylamak.
         d) Bakanlık ve Müdürlük ile ilgili olan her türlü muhabere ve yazışmaları yürütmek.
         e) Özel hastanenin personel ve hizmet kalite standartlarını korumak ve geliştirmek.
@@ -362,9 +360,9 @@
         (5) Özel hastaneler, acil sağlık hizmetlerini düzenleyen ilgili diğer mevzuata da uymak zorundadır.
         (6) Özel hastanelerin acil ünitesinde günün her saatinde tabip bulundurulması zorunludur. Hastanenin ruhsata esas uzmanlık dallarında acil tıp
 uzmanı bulunması halinde bu tabip acil serviste görevlendirilir. Acil tabibi tarafından yapılacak davete ilgili dal uzmanının ve diğer personelin uymaları
- zorunludur. Normal çalışma saatleri dışında faaliyet gösterilen uzmanlık dallarındaki uzman tabipler icapçı olarak görev yaparlar. Ancak mevcut kadrolu
 veya kadro dışı geçici çalışan uzman tabipler, mesul müdürce düzenlenen nöbet listesine göre acil ünitesinde nöbet tutabilir. Hastanede kadro dışı
+ çalışan tabipler de nöbet tutabilir. Acil serviste nöbet tutan tabiplerin üstlerinde fotoğrafları ve “Nöbetçi Tabip” yazılı mesul müdür tarafından
 düzenlenmiş kimlik kartı bulunmak zorundadır. Nöbetçi tabip listesi haftalık olarak hazırlanır ve mesul müdür tarafından onaylanarak dosyalanır.
         (7) Belirli bir uzmanlık dalında faaliyet gösteren hastanelerin acil üniteleri sürekli olarak kullanıma hazır halde bulundurulur ve faaliyet
@@ -387,9 +385,10 @@
 üstü tabipler, sözleşme yaptığı hastanelerin birinde bu kadro sınırlamasına tabi tutulmaz.)
         (3) Muayenehanenin bulunduğu ilde ilgili uzmanlık dalında sözleşme yapılabilecek uygun özel sağlık kuruluşu bulunmadığı hallerde talep edilmesi
 halinde Bakanlığın uygun görüşüyle yakın bir ildeki özel hastane ile sözleşme yapılabilir.
-        ( 4) (Danıştay Onuncu Dairesinin 31/12/2025 tarihli ve E.:2025/1585 sayılı kararı ile yürütmesi durdurulan fıkra;Sözleşme
-imzalanan hastanede muayenehaneden gelen hastaya öngörülen müdahalenin yapılamadığı durumlarda tabibin istediği başka bir
-hastanede bu işlemin yapılması için ilgili il sağlık müdürlüğü tarafından vaka bazlı özel izin verilebilir.)
+        (4) (Değişik:RG-29/8/2026-33355) Sözleşme imzalanan hastanede muayenehaneden gelen hastaya öngörülen müdahalenin yapılamadığı
+durumlarda tabibin istediği başka bir hastanede bu işlemin yapılması için ilgili il sağlık müdürlüğü tarafından vaka bazlı özel izin verilebilir. Bu fıkra
+kapsamındaki iznin istenmesi, izin talebinin değerlendirme kriterleri ve değerlendirme süreci ile iznin acil hallerde, mesai saatleri dışında ve tatil
+günlerinde uygulanmasına ilişkin usul ve esaslar Bakanlıkça belirlenir.
         (5) Muayenehane tabibiyle yapılan sözleşmenin taraflarca imzalanmış nüshası imzalandığı tarihten itibaren en geç 5 iş günü içinde özel hastane
 tarafından EKİP sistemine yüklenerek Müdürlüğe gönderilir. Sözleşmenin sona erdiği hallerde bu durum sözleşme bitiş tarihinden itibaren en geç 5 iş
 günü içinde Müdürlüğe bildirilir.
@@ -435,8 +434,8 @@
         Tıbbi kayıt ve arşiv
         MADDE 29- (1) Özel hastanelere başvuran hastalara ait veriler elektronik ortamda kaydedilir. Verilerin kaydedilmesi ve arşivlenmesi
 hususlarında Bakanlığın kayıt tescil sisteminde kayıtlı bir sağlık bilgi yönetim sistemi kullanılması zorunludur.
--         (2) Özel hastaneler tarafından kayıt altına alınan kişisel sağlık verileri, 24/3/2016 tarihli ve 6698 sayılı Kişisel Verilerin Korunması Kanunu ile
+        (2) Özel hastaneler tarafından kayıt altına alınan kişisel sağlık verileri, 24/3/2016 tarihli ve 6698 sayılı Kişisel Verilerin Korunması Kanunu ile
+ ikincil düzenlemelere ve 3359 sayılı Kanunun ek 19 uncu maddesine uygun bir şekilde Bakanlıkça belirlenen usul ve esaslara göre işlenerek merkezi
 sağlık veri sistemine aktarılır. Bakanlık tarafından kurulan kayıt ve bildirim sistemine ve Bakanlıkça yapılacak diğer iş ve işlemlere esas olmak üzere,
 istenilen bilgi ve belgelerin gönderilmesi zorunludur.
@@ -508,8 +507,8 @@
 dışındaki diğer isim ve ünvanları ile faaliyet izin belgesinde belirtilen uzmanlık dalları haricinde başka uzmanlık dalı yazılamaz.
          (2) Özel hastane isimlendirilmesinde; yanlış anlaşılma ve karışıklığın önlenmesi amacıyla, hangi ilde olduğuna bakılmaksızın ülke genelindeki
 üniversite, kamu sağlık kurum veya kuruluşları ile özel sağlık kuruluşlarının isimleri aynen veya çağrıştıracak şekilde kullanılamaz.
--       (3) Hastanelere kurumun amacına uygun, kamu düzeni ve genel ahlaka aykırı olmayan ve Bakanlıkça onaylanan isimler verilir.
+         (3) Hastanelere kurumun amacına uygun, kamu düzeni ve genel ahlaka aykırı olmayan ve Bakanlıkça onaylanan isimler verilir.
+        Dini ödevlerin yapılması
        MADDE 39- (1) Özel hastanelerde, hastaların dini gereklerini yerine getirebilecekleri mekanlar ayrılır. Agoni halindeki hastaların dini
 hizmetlerden faydalanmaları için Hasta Hakları Yönetmeliğinin ilgili maddesi uyarınca gereken tedbirler alınır.
@@ -581,8 +580,8 @@
        a) Gerekli olan uzman tabip, tıbbi donanım ile asgari sağlık personelinin eksik olduğunun tespiti halinde, ilgili birimde eksiklik giderilinceye kadar,
        b) Hastaların tedavisini olumsuz etkileyecek, toplumun veya sağlık hizmeti alanların sağlığını olumsuz etkileyeceği düşünülen durumların ortaya
 çıkması halinde bu durum ortadan kaldırılana kadar,
--         c) 40 ıncı maddenin üçüncü fıkrasında belirtilen durumların tespiti halinde gerekli idari tedbirler alınıncaya kadar,
+       c) 40 ıncı maddenin üçüncü fıkrasında belirtilen durumların tespiti halinde gerekli idari tedbirler alınıncaya kadar,
+          ilgili birimin faaliyeti durdurulur. Bu fıkranın (b) ve (c) bentleri uyarınca yapılacak faaliyet durdurma işleminde Bakanlığın uygun görüşü aranır.
          Hastanenin tamamında faaliyetin geçici olarak durdurulması
          MADDE 45- (1) Özel hastanenin tamamında Bakanlığın kararı üzerine Valilikçe;
@@ -654,8 +653,8 @@
          d) Klinik konukevlerinin yatak ücretleri Bakanlıkça onaylanır.
          (6) Geriatrik tedavi merkezi; yaşlılıkta gelişen hastalıkların tanı ve tedavisi, rehabilitasyonu ile beraberinde bakım hizmetlerinin sunulmasına
 yönelik olarak kurulan sağlık kuruluşlarıdır. Bu kuruluşun kadro ve kapasitesi, Bakanlıkça yapılan planlama çerçevesinde belirlenir. Yaşlı hastalara
- multidisipliner yaklaşımla hizmet sunulabilmesi amacıyla, hastane bünyesinde veya hastaneye bağlı olarak kurulabilir. Bu Yönetmelikte belirlenen
+ şartlara uygun binalarda hizmet sunabilir.
         (7) Bu maddede hüküm bulunmayan hallerde öncelikle ilgili mevzuat hükümleri uygulanır. Bakanlık uygulamaya yönelik hususlarla ilgili alt
 düzenlemeler yapmaya yetkilidir.
@@ -708,9 +707,10 @@
 Yönetmeliğinin geçici 1 inci maddesi kapsamında Bakanlıktan ön izin almış özel hastanelerden beş yıllık ruhsatlandırma süresi sonunda ruhsat
 işlemlerini tamamlayamayanlara, 31/12/2028 tarihine kadar ruhsatlandırma işlemlerini tamamlamak amacıyla ek süre verilir.
         Muayenehanesini kapatan tabiplerin durumu
-        GEÇİCİ MADDE 9-(1) 1/1/2025 tarihi itibarıyla faal muayenehanesi olup başka bir sağlık kuruluşunda çalışmayan tabiplerin
-muayenehanelerini kapatması halinde aynı ildeki en fazla iki özel hastanede kadro dışı geçici olarak çalışmasına izin verilir. Bu tabiplerin yeniden
-muayenehane açmaları halinde bu madde kapsamında verilen kadro dışı geçici çalışma hakkı sona erer.
+        GEÇİCİ MADDE 9- (1) 1/1/2025 tarihi itibarıyla faal muayenehanesi (Değişik ibare:RG-29/8/2026-33355) olan tabiplerin
+muayenehanelerini kapatması halinde (Değişik ibare:RG-29/8/2026-33355) muayenehanesini kapattığı ildeki en fazla iki özel hastanede kadro dışı
+geçici olarak çalışmasına izin verilir. Bu tabiplerin yeniden muayenehane açmaları halinde bu madde kapsamında verilen kadro dışı geçici çalışma hakkı
+sona erer.
         Tabip kadrolarının devri
         GEÇİCİ MADDE 10-(1) Bu maddenin yürürlüğe girdiği tarih itibarıyla ruhsatlı ve faal olan özel hastaneler maddenin yürürlük tarihinde boş
 olan tabip ve uzman tabip kadrolarını Bakanlık planlamalarına uygun olmak kaydıyla 31/12/2025 tarihine kadar başka bir özel sağlık kuruluşuna
@@ -727,18 +727,26 @@
 asgari fiziki kriterlerden Bakanlıkça muaf tutulabilir. Bu hastanelerin yeni binaya taşınmaları halinde asgari fiziki kriterlere uyması zorunludur.
         Kadro ve yatak ilavesi
 -       GEÇİCİ MADDE 14-(1) Bu Yönetmeliğin geçici 2 nci maddesinde sayılan özel hastaneler ile mevcut binası 18/3/2018 tarihli ve 30364 sayılı
+        GEÇİCİ MADDE 14-(1) Bu Yönetmeliğin geçici 2 nci maddesinde sayılan özel hastaneler ile mevcut binası 18/3/2018 tarihli ve 30364 sayılı
 mükerrer Resmî Gazete’de yayımlanan Türkiye Bina Deprem Yönetmeliğine göre riskli olduğuna dair deprem raporu sunan özel hastanelerden,
 taşınmak ve/veya yerinde yeni bina yapmak isteyenlere ruhsatlandırma aşamasında Bakanlıkça belirlenen uzmanlık dallarında uzman tabip kadro
 sayısına yüzde 10 ve yatak sayısına yüzde 30 ilave edilir. Bu hastanelerin, bulundukları ilden başka ile taşınmak istemeleri halinde bir sefere mahsus
 Bakanlıkça planlama kriterleri çerçevesinde uygun görülen illere taşınmasına izin verilir.
-       Yürürlük
-       MADDE 53- (1) Bu Yönetmeliğin;
-       a) 16 ncı maddesinin altıncı fıkrası ve 22 nci maddesi 1/1/2026 tarihinde,
-       b) Geçici 12 nci maddesi 31/12/2024 tarihinden geçerli olmak üzere yayımı tarihinde,
-       c) Diğer hükümleri yayımı tarihinde,
-       yürürlüğe girer.
-       Yürütme
-       MADDE 54- (1) Bu Yönetmelik hükümlerini Sağlık Bakanı yürütür.
+        Fiziki şartlardan muafiyet
+                                                                 [1]
+        GEÇİCİ MADDE 15- (Ek:RG-29/8/2026-33355)
+        (1) 30/1/2025 tarihinden önce ruhsatlandırılmış özel hastaneler, bu Yönetmelikte yer alan fiziki şartlardan muaftır. Bu muafiyet, 30/1/2025 tarihi
+itibarıyla özel hastanelerce yapılan değişiklikler ve geçici 2 nci madde kapsamında yer alan özel hastaneler için uygulanmaz.
+        Yürürlük
+        MADDE 53- (1) Bu Yönetmeliğin;
+        a) 16 ncı maddesinin altıncı fıkrası ve 22 nci maddesi 1/1/2026 tarihinde,
+        b) Geçici 12 nci maddesi 31/12/2024 tarihinden geçerli olmak üzere yayımı tarihinde,
+        c) Diğer hükümleri yayımı tarihinde,
+        yürürlüğe girer.
+        Yürütme
+        MADDE 54- (1) Bu Yönetmelik hükümlerini Sağlık Bakanı yürütür.
 
-      Ekleri için tıklayınız
+         Ekleri için tıklayınız
+
+[1]
+      Bu değişiklik 30/1/2025 tarihinden geçerli olmak üzere yayımı tarihinde yürürlüğe girer.
